packages/amplify_core/lib/src/types/storage/download_file/download_file_request.dart

```diff
@@ -31,7 +31,7 @@ class DownloadFileRequest {
   Map<String, Object?> serializeAsMap() => {
         if (options != null) 'options': options!.serializeAsMap(),
         'key': key,
-        'local': local,
+        'path': local.absolute.path,
```

Comment:
[Native code ](https://github.com/aws-amplify/amplify-flutter/blob/aa8fd3b3bf3b3420a0e7ac60b1867e64550910d2/packages/storage/amplify_storage_s3_android/android/src/main/kotlin/com/amazonaws/amplify/amplify_storage_s3/types/FlutterDownloadFileRequest.kt#L28)seems to require `path` and not `local`, right?

Reply:
Yes it looks like that the currently implementation is wrong, as it doesn't respect API consumer provided `File` spec... 😅